lib/engine/src/trap/frame_info.rs

```diff
@@ -74,10 +75,11 @@ impl ModuleInfoFrameInfo {
     /// Gets a function given a pc
     fn function_info(&self, pc: usize) -> Option<&FunctionInfo> {
         let (end, func) = self.functions.range(pc..).next()?;
-        if pc < func.start || *end < pc {
-            return None;
+        if func.start <= pc && pc <= *end {
```

Comment:
I somehow think the `.contains` syntax makes it harder to read?

Reply:
I kept this syntax because it's closer to:
```
if func.start <= pc <= *end {
```

Reply:
Drive by comment, but I am a fan of only using < and chained comparisons: https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#comparisons

Reply:
That's interesting! I usually agree with that type of style advice, but I actually find it much harder to read. My reason is that when doing a bounds check like X is between Y and Z, X is the subject, so you should focus on X. To me it's way more natural to say "X is greater than Y but less than Z" than it is to say "Y is less than X and X is less than Z". Maybe it's because I read it like a sentence rather than a mathematical expression.

Outside of the context of a bounds check though, I can get behind consistent use of a comparison operator.